Plugins/Wox.Plugin.Program/Programs/Win32.cs

```diff
@@ -33,12 +33,10 @@ public class Win32 : IProgram
 
         private int Score(string query)
         {
-            var score1 = StringMatcher.FuzzySearch(query, Name).ScoreAfterSearchPrecisionFilter();
-            var score2 = StringMatcher.ScoreForPinyin(Name, query);
-            var score3 = StringMatcher.FuzzySearch(query, Description).ScoreAfterSearchPrecisionFilter();
-            var score4 = StringMatcher.ScoreForPinyin(Description, query);
-            var score5 = StringMatcher.FuzzySearch(query, ExecutableName).ScoreAfterSearchPrecisionFilter();
-            var score = new[] { score1, score2, score3, score4, score5 }.Max();
+            var nameMatch = StringMatcher.FuzzySearch(query, Name);
+            var descriptionMatch = StringMatcher.FuzzySearch(query, Description);
```

Comment:
@jjw24 Why did we remove the Score for pinyin support?

Reply:
It's moved to within the fuzzy search method in string matcher since it is called on all plugin searches.